### ISSUE TITLE
R CMD check failed does not end build

### ISSUE THREAD

**Issue by jstockwin** [travis-ci/travis-ci#7893]
@craigcitro, @eddelbuettel and @hadley

I'm finding that when my R check fails, it simply stops outputting rather than explicitly ending the job. After 10 minutes Travis picks up on the lack of output and fails the build, but obviously this adds an unnecessary 10 minutes to the build time.

See https://travis-ci.org/jstockwin/EpiEstimApp/jobs/241731061#L1286 for an example.

**Comment by eddelbuettel** [comment]
You may be looking at old docs; my name was removed from the support list a looooong time ago. I ise and maintain an unofficial renegade fork of the older scheme and can help with that; for your current troubles I am unable to offer assistance as I do not use it myself.  So I will unsubscribe here.

**Comment by jstockwin** [comment]
True, sorry! Copied the above list from another issue I came across while trying to google the above. 
I guess I should add @jimhester to this.

**Comment by pierucci** [comment]
Same problem here.
